pages/common/netlify.md

```diff
@@ -0,0 +1,20 @@
+# netlify
+
+> Deploy sites and configure continuous deployment to the Netlify platform.
+> More information: <https://www.netlify.com/docs/cli/>.
+
+- Login to a Netlify account:
+
+`netlify login`
+
+- Create a new deploy from the contents of a directory:
```

Comment:
I'm not sure this sounds right. What about this?
```suggestion
- Create a new deployment from the contents of a directory:
```

...or this?
```suggestion
- Deploy the contents of a directory to Netlify:
```

Reply:
That's what I said in my review comment too ;)